CHANGELOG.md

```diff
@@ -1,7 +1,288 @@
+  - **Breaking change**: alpaka platforms are now full objects instead of types #1988 #2051
+  - `operator<<(std::ostream&, WorkDivMembers const&)` is now a `friend` of `alpaka::WorkDivMembers` instead of a method #1829
+  - **Potentially breaking change**: Switched several view-related methods from `ALPAKA_FN_HOST_ACC` to `ALPAKA_FN_HOST` #1826
+  - Accelerators' copy/move constructors and assignment operators are now explicitly `delete`d #1825
+  - `alpaka::test::allocAsyncBufIfSupported` was moved into the general `namespace alpaka` #1782
+  - Removed unnecessary attribute `ALPAKA_FN_HOST_ACC` from defaulted functions #1761
+  - The `UniformCudaHip` types are now templated on traits-like `struct`s which encapsulate the CUDA or HIP API #1665
+- General behavioural changes:
+  - Improved handling of CMake generator expressions #2146
+  - Improved detection of C++20 features #2138
+  - Simplified internals of `alpaka_add_{executable,library}` #2072 #2082
+  - **Breaking change**: Removed dummy atomics from memory fence implementations. Users now need to guarantee correctness themselves #2071
+  - In debug mode MSVC will use the `/Od` optimization level #1977
+  - In debug mode clang-based compilers will explicitly use the `-O0` optimization level #1977
+  - In debug mode `g++` will use the `-Og` optimization level #1977
+  - `-Werror` and its MSVC equivalent `/WX` are no longer enabled by default when `BUILD_TESTING` is set to `ON` #1977
+  - A platform's internal `std::vector` containing the `alpaka::Device`s now reserves the necessary memory before initialization #1926
+  - **Potentially breaking change**: `ALPAKA_FN_INLINE` now enforces inlining for platforms other than CUDA and HIP #1918
+  - Replaced `alpaka::core::ConcurrentExecPool` with `alpaka::core::CallbackThread` in all queue implementations #1870
+  - If no back-end is enabled, alpaka automatically selects the serial back-end for examples and test cases #1843
+  - On Linux platforms, the free global memory is now determined by a call to `sysconf(_SC_AVPHYS_PAGES)` instead of querying `/proc/sysinfo` #1776
+  - **Potentially breaking change**: Changed CMake's look-up of MSVC's runtime libraries (see [here](https://cmake.org/cmake/help/v3.22/policy/CMP0091.html) for an in-depth explanation) #1751
+  - Unified `alpaka::{memcpy,memset}`'s internal `static_assert`s #1748
+  - `alpaka::core::aligned{Alloc,Free}` now internally use aligned `new`/`delete` instead of OS-specific APIs #1689
+- CUDA/HIP back-end changes:
+  - `nvcc` now makes correct use of `--Werror` and more CUDA-related warnings #2135
+  - Unified `ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK` macros #2090
+  - Made some internal constants `constexpr` #2063
+  - The CUDA/HIP back-ends will now always use `std::size_t` for internal pitch calculations #2056
+  - **Breaking change**: clang as CUDA compiler will only work in `Release` build mode #2027
```

Comment:
That is a workaround on our side, and not a change, isn'it it? Maybe call it a "breaking workaround", or something else.

Reply:
This one is tricky. The underlying issue is a bug on clang's side. Our "workaround" is that we simply don't officially support clang-cuda in `Debug` mode. But "breaking workaround" sounds weird to me so I'd prefer "change" here.

Reply:
Alright.